packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1322,6 +1322,50 @@ const messages = {
       external: 'Ekstern ressurs',
     },
   },
+  gloss: {
+    examples: 'Eksempler',
+    transcriptions: {
+      traditional: 'tradisjonell',
+      pinyin: 'pinyin',
```

Comment:
Basert på skjermleser-behov
```suggestion
      traditional: 'Tradisjonell skrivemåte',
      pinyin: 'Pinyin',
```